sway/input/cursor.c

```diff
@@ -564,6 +565,7 @@ static struct sway_binding* get_active_mouse_binding(
 				cursor->pressed_button_count != (size_t)binding->keys->length ||
 				release != (binding->flags & BINDING_RELEASE) ||
 				!(click_region & binding->flags) ||
+				(on_workspace && click_region != binding->flags) ||
```

Comment:
Should `click_region != binding->flags` be a bitwise operation instead?

Reply:
`binding->flags` must be `BINDING_TITLEBAR | BINDING_BORDER | BINDING_CONTENTS` (aka whole-window) for the bindings to trigger on a workspace. Earlier in the function, `click_region` is set to that value so a simple equality check should be sufficient

Reply:
Actually, nevermind. It does need to be bitwise since there are other non-region flags

Reply:
Changed